tests/foreman/installer/test_installer.py

```diff
@@ -1299,22 +1297,14 @@ def test_positive_foreman_module(target_sat):
 
     :CaseLevel: System
 
-    :expectedresults: Foreman RPM and SELinux module versions match
+    :expectedresults: Foreman RPM and SELinux module are both present on the satellite
     """
     rpm_result = target_sat.execute('rpm -q foreman-selinux')
     assert rpm_result.status == 0
 
     semodule_result = target_sat.execute('semodule -l | grep foreman')
     assert semodule_result.status == 0
 
-    # Sample rpm output: foreman-selinux-1.7.2.8-1.el7sat.noarch
-    version_regex = re.compile(r'((\d\.?)+[-.]\d)')
-    rpm_version = version_regex.search(rpm_result.stdout).group(1)
-    # Sample semodule output: foreman        1.7.2.8
-    semodule_version = version_regex.search(semodule_result.stdout).group(1)
-    rpm_version = rpm_version[:-2]
-    assert rpm_version.replace('-', '.') == semodule_version
-
```

Comment:
@Griffin-Sullivan I debugged this test a long ago, iirc, this selinux module is on el7 sat and not el8, so can we update this block to run only when it is el7, wdyt?

Reply:
Yeah we could. I originally didn't plan on cherry-picking this to 6.11. Are other still cherry-picking back to this branch? Are we running this automation ever? If so, I can add the conditional here.

Reply:
Yes, we're actively maintaining 6.11.z, and soon there will be a snap :)

Reply:
Yep just saw that! Gonna update this to run on <= RHEL7.